tests/kernel/install_ltp.pm

```diff
@@ -270,7 +274,10 @@ sub run {
 
     upload_logs('/boot/config-$(uname -r)', failok => 1);
 
+    suseconnect_registration;
+    command_register(get_required_var('VERSION'), 'sdk'); # FIXME: debug
```

Comment:
'sdk' exists on SLE11/SLE12 but not onSLE15

Reply:
Thx! I'll remove this code and evaluate, whether works `SCC_ADDONS=sdk` `ADDONS=sdk`. Or do you know a way how to add SDK repo for all versions?